piplicenses.py

```diff
@@ -220,6 +229,26 @@ def get_pkg_info(pkg):
             continue
 
         pkg_info = get_pkg_info(pkg)
+
+        license_name = pkg_info['license']
+        if fail_on_licenses and license_name in fail_on_licenses:
```

Comment:
In packages that adopt a dual license, the `license_name` is expressed as a comma-separated list. A comparison with an exact match may be made through.

e.g.) 
```sh
# Packages distributed under dual licenses
$ pip show -v keyring | grep "License :: OSI Approved"
  License :: OSI Approved :: Python Software Foundation License
  License :: OSI Approved :: MIT License

# That's comma-separated output 
$ pip-licenses | grep keyring 
 keyring             21.4.0     Python Software Foundation License, MIT License

# Just specify "Python Software Foundation License" and it went through
$ pip-licenses --fail-on="Python Software Foundation License"
$ echo $?
0
```

Is this as expected?

If it is as expected, it would be nice to have it described in more detail in the `README` documentation.

Reply:
Indeed, dual licenses is an issue we have using https://www.npmjs.com/package/license-checker also where `MIT or GPL` is distinct from `MIT`.

This is less of an issue for `allow-only` as you can add `MIT or GPL` and `MIT` when you see the check is failing. It means you are forced to approve the new license combination. This is the form I need most.

However for `fail-on` this may be more of an issue. For example you may be trying to specifically prevent `GPL` and as in your example, this would not match if there was another license also attached. I agree that a stern warning in the README would be a good start and then a possible improvement could come later, I'll do that now ;)